### PR TITLE
html 

### PR DESCRIPTION
**Pull request by thetechtakeover**
added html file to origin

### CODE REVIEW

**Comment by nojocode** [comment]
I am going to add my parts to the code tonight

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css href="main.css">
```
Comment (nellarro) [inline index.html:6]: Looks great!

**Review by nellarro — APPROVED** [review]
🎉 Great job

**Comment by nellarro** [comment]
Fixes #1

**Pull request merged**